doc/dbus_api.md

```diff
@@ -288,3 +288,42 @@ when the question is answered.
 - Attempt -> unsigned 32-bit integer (r)
   Current attempt to decrypt the device. This value is useful for clients to know if the very same
   question is asked again (i.e., when the provided password did not work).
+
+
+## Status
+
+The main object of a service implements the following interface:
+
+### org.opensuse.DInstaller.Status1
+
+#### Properties
+
+- All -> array(dict(string, variant)) (r)
```

Comment:
Perhaps "All" is too generic. However, I would say we are safe, as all `org.freedsktop.DBus.Properties` methods get an interface name as the first argument, so it should not be a problem. I wonder, however, how D-Feet is going to show that property.

Reply:
Oh, as @joseivanlopez pointed out the properties are shown under their own interface. So that's fine.